src/embeddings/embeddings_api.cpp

```diff
@@ -121,6 +123,49 @@ std::variant<EmbeddingsRequest, std::string> EmbeddingsRequest::fromJson(rapidjs
         }
     }
 
+    if (useTokenizeEndpoint) {
```

Comment:
it would be good to create shared function validate_tokenizer_request because it could be shared with multiple calculators with a tokenizer.